File: schemas/entregador.py
from pydantic import BaseModel, Field
from typing import Optional
from datetime import datetime

class EntregadorCreate(BaseModel):
    nomEntrg: str
    numCNHEntrg: str
    dscPlacaVeiculoEntrg: str
    dscTipoVeiculoEntrg: str
    numTelefoneEntrg: str

class EntregadorResponse(BaseModel):
    id: str
    nomEntrg: str
    numCNHEntrg: str
    dscPlacaVeiculoEntrg: str
    dscTipoVeiculoEntrg: str
    numTelefoneEntrg: str

File: frontend/src/types/entregador.ts
export interface Entregador {
  idEntrg: string;
  nomEntrg: string;
  numCNHEntrg: string;
  dscPlacaVeiculoEntrg: string;
  dscTipoVeiculoEntrg: string;
  numTelefoneEntrg: string;
}

export interface EntregadorCreate {
  nomEntrg: string;
  numCNHEntrg: string;
  dscPlacaVeiculoEntrg: string;
  dscTipoVeiculoEntrg: string;
  numTelefoneEntrg: string;
}